app/models/notification.rb

```diff
@@ -1,4 +1,46 @@
 class Notification < ApplicationRecord
   belongs_to :bot
   alias_attribute :type, :notification_type
+  after_initialize :init
+
+  scope :pending_human_override_for_session, ->(session_uuid) {
+    Notification.where(session_uuid: session_uuid, resolved: false, notification_type: 'human_override')
+  }
+
+  def init
+    self.uuid ||= SecureRandom.uuid
+  end
+
+  def bot_forwarding_messages!(value)
+    raise 'bot_forwarding_messages only supported for a human_override notification' if notification_type != 'human_override'
+
+    data['bot_forwarding_messages'] = value
+  end
+
+  def bot_forwarding_messages?
+    data['bot_forwarding_messages'] || false
```

Comment:
In ruby there is no need for the `|| false` clause as `nil` is a "falsey" value. `|| false` works because of that.